### ISSUE TITLE
Motion Sensor 2 doesn't accept LED color change (or LED off)

### ISSUE THREAD

**Issue by vk900** [athombv/com.fibaro#175]
**What is your Homey version (Settings → System)?**
1.5.3

**What did you try to achieve?**
Change color of LED within the Motion Sensor 2
(read about other people trying to turn of the LED completely, without result either)
Also tried to change the wake-up interval time to a low value to ensure that settings would get pushed in a certain timefraime.

**What was the result?**
Settings within the Motion Sensor 2 stay unchanged (even after multiple days, indicating that this hasnt got anything to do with a wake-up interval)

**What did you expect as result?**
Settings to be pushed towards the Motion Sensor 2

**Any other remarks**
Mostlikely none of the other settings are changable either

**Please attach a Homey System log (Settings -> System -> Send Log**)

**Comment by RobinBol** [comment]
Hi, could you confirm that changing the settings works when waking up the device manually directly after/before changing (!) and saving the settings in Homey?

**Comment by vk900** [comment]
will try tonight, just to be sure: how do i manually wake the device?

**Comment by RobinBol** [comment]
Depends on the device, Fibaro motion sensor is clicking the button thrice I think.

**Comment by vk900** [comment]
Okay i'll check it out and let you know.

_a bit offtopic but I yesterday evening i was trying to change configuration settings of a Fibaro KeyFob while waking the device following the Homey-procedure but no luck with that.. Somehow pushing configuration settings to Fibaro battery-backed devices isn't real stable.._

**Comment by RobinBol** [comment]
The waking procedure of the keyfob is a bit tricky, and probably the cause of the settings not applying. But if you have unrelated issues, please create a separate issue for that.

**Comment by vk900** [comment]
Will check it out again and if needed post another issue. I know it's tricky and tried 10 times exactly following the "good" procedure... next update from me will be on the Motion Sensor. Thanks :)

**Comment by vk900** [comment]
Doesn't work..

So what I did:

1)
- Opened the Motion Sensor 2's config screen
- Pressed button 3 times, led stays on after doing that
- Changed a setting, pressed Save
- No luck
 
2)
- Pressed button 3 times, led stays on after doing that
- Opened the Motion Sensor 2's config screen
- Changed a setting, pressed Save
- No luck

**Comment by RobinBol** [comment]
What did the pop up say?

**Comment by vk900** [comment]
Device settings have been saved

**Comment by RobinBol** [comment]
Ok, and after say 5 minutes the led color did not change accordingly?

You can try to use the raw configuration parameters in de device settings, just check the manual for the correct values, if that works (when waking up the device), then there is a bug in the settings parser.

**Comment by vk900** [comment]
Nope... I checked to see that i didn't got the message saying "at next wakeup... bla bla bla"
Okay let me check

**Comment by vk900** [comment]
![capture](https://user-images.githubusercontent.com/32387459/31500196-150965ae-af67-11e7-9045-2fef40173dfa.PNG

not sure how to convert that to a raw value in the device settings

**Comment by vk900** [comment]
it gets even more interesting cause when I'm troubleshooting this (so putting the motion sensor in learn and pushing new settings), after that none of my Fibaro modules work anymore with Homey. (so I cannot switch dimmers and/or single switches).

Only after a reboot this functionality comes back again.

_(This is the same I had yesterday while trying to push settings to the KeyFob so I think the problem is even bigger and wrecks the complete Fibaro-stack)_

**Comment by RobinBol** [comment]
80,1,0 should turn off the led completely

**Comment by vk900** [comment]
Let me run a complete scenario again starting with a Homey boot and a functional Fibaro stack as I noticed that it did accept 1 of my color changes..... man, what a mess :(

**Comment by vk900** [comment]
lol, when I now rebooted the Homey i heared multiple single in-wall switches switch so it looks like something is stalling something which comes back to live whenever the Homey get a reboot..

**Comment by vk900** [comment]
I think I started off with an inconsistent Homey state due to my troubleshooting the KeyFob yesterdan and apparently wrecking the Fibaro-stack.

This is what I did now and this works:

-reboot Homey
-Verified I can turn on/off single fibaro switches and wall-plug = okay
-Opened the Motion Sensor 2's config screen
-Pressed button 3 times, led stays on after doing that
-Changed a setting, pressed Save (you have very little time for this before the LED goes off again)
-Got message that settings were saved (no message about "at next wake-up settings will be pushed")
-Verified I can turn on/off single fibaro switches and wall-plug = okay

I'll now try changing KeyFob settings and will report back findings in another issue if applicable.
At least I know how to f*ckup the Fibaro-stack and making it unusable :|

**Comment by RobinBol** [comment]
Could you report exactly what setting you are trying to change to what value? I have been trying to reproduce your issue here with a FGMS-001-PLUS, and it works as expected.

**Comment by mruiter** [comment]
This sounds like the zwave bug in 1.5.x reports get in send does not work.
motion sensor seems ok in this case only config changes does not work.

time for a fix

for now a pull the plug will most likely fix youre issue
please confirm . 


Verstuurd vanaf mijn iPad

**Comment by vk900** [comment]
Are you waiting for my input Robin? (as I finally got it to work... i did create another issue for the KeyFob thingie..)

**Comment by RobinBol** [comment]
Ok, thanks, I will close this issue then. @mruiter that is going to be investigated asap
